src/extensions/treegrid/bootstrap-table-treegrid.js

```diff
@@ -8,7 +8,9 @@ $.extend($.fn.bootstrapTable.defaults, {
   treeShowField: null,
   idField: 'id',
   parentIdField: 'pid',
-  rootParentId: null
+  rootParentId: null,
+  //Use this property name to indicate whether a child node exists，default‘haveChild’
+  haveChildField:'haveChild'//用此属性名表示是否存在子节点，默认‘haveChild’
```

Comment:
Could we use a bool instead of a string?

Reply:
No, the function of this field is to control whether there are child nodes in an object。
it refers to the name of the field, not the value of the field。
for example， You can refer to the last picture I sent above。